lambda-events/src/main/scala/feral/lambda/events/DynamoDBStreamEvent.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.lambda
+
+import io.circe.Decoder
+import io.circe.Json
+
+final case class AttributeValue(
+    b: Option[String],
+    bs: Option[String],
+    bool: Option[Boolean],
+    l: Option[List[AttributeValue]],
+    m: Option[Map[String, AttributeValue]],
+    n: Option[String],
+    ns: Option[List[String]],
+    nul: Option[true],
```

Comment:
We're going to drop 2.12 so this should be fine. Literal types yay!

Reply:
tbh this could be `nul: Boolean`, but the decoder will be nastier :(